dist/script_offline.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+  const session = new scribble.Session();
+
+  session.createChannel({
+    instrument: 'PolySynth',
+    clips: [
+      { pattern: '[xx]', notes: 'C3 Cm-3' },
+      { pattern: '[x-]', notes: 'C3 Cm-3' },
+      { pattern: 'xx', notes: 'C3 Cm-3' },
+    ],
+    offlineRendering: true,
```

Comment:
Should we add `offlineRendering` in here as well: https://github.com/scribbletune/scribbletune/blob/master/src/typings.d.ts#L156-L159

Reply:
May not be needed on second thoughts.